model/labels/labels_dedupelabels_test.go

```diff
@@ -48,3 +48,25 @@ func TestVarint(t *testing.T) {
 	}
 	require.Panics(t, func() { encodeVarint(buf[:], len(buf), 1<<29) })
 }
+
+func TestByteSize(t *testing.T) {
+	for _, testCase := range []struct {
+		lbls     Labels
+		expected int
+	}{
+		{
+			lbls:     FromStrings("__name__", "foo"),
+			expected: 4,
+		},
```

Comment:
I have a suggested pattern to avoid all this repetition, at #16807

Reply:
Hi, will you update to follow the pattern?
I.e. I expect an extra set of expected sizes (or maybe not, if the sizes come out the same)